cmd/root.go

```diff
@@ -123,6 +114,11 @@ func setupLogger(atmosConfig *schema.AtmosConfiguration) {
 	}
 
 	log.SetOutput(output)
+	if _, err := logger.ParseLogLevel(atmosConfig.Logs.Level); err != nil {
+		//nolint:all
```

Comment:
Why are we no linting here?

Reply:
We get error that we should only exit in main and init. If we ignore that we get the error that we should wait for the the defer function before exiting.

Reply:
Please add a comment with that explanation. Also, this undermines defer and why we should (not now) fix it.